.travis.yml

```diff
@@ -18,9 +18,12 @@ script:
   - 'if [ "${TEST-}" = true ]; then npm run tests-only ; fi'
   - 'if [ -n "${KARMA-}" ]; then npm run tests-karma ; fi'
   - 'if [ -n "${COVERAGE-}" ] && [ "${TRAVIS_BRANCH-}" = "master" ]; then npm run cover; cat ./coverage/lcov.info | ./node_modules/.bin/coveralls ; fi'
+  - 'if [ -n "${HAPPO-}" ]; then npm install --no-save happo-plugin-puppeteer && INSTALL_CMD="npm install && npm install --no-save happo-plugin-puppeteer" npm run happo-ci ; fi'
 env:
   global:
     - TEST=true
+    - secure: "lv5bnKRHd3A0tUequn7CrpKLkaZjVc1YmY/MzvX53wjqIwP4llpIil9XqeVYoG/VDGcf810XktZe2QeAuFUTXIZhPGQRXcWsbKbHVNmKSFQFDtXqEpB9NGw4NBCytK2ZLVXhRfrIcHm6Ltr0CSa+mO/KHtk9bE/on/Itjc0aB5b7Y1gVVHTfyZ3QOjIsPUenLRsY1tn1rUJ0meFlogPFIBbcnK+GPDL2oHsiVIOjCfBauIcRlNP7XZggFhZbHARZG7bTDr+YBuybAIBCVnex5Av7ho5EHu8FTjNrGXZyKF5IOMjqJla8UvOAoEnNe5k98O+Wx9ehlObUYGcr9szV9zGHvp0Wo+sflHdWYy5IOQAp+AQqjIz8J18ia6q0Zm9TBL5h2OEx/XwKzp4Q+F0XeTL7USnsfLuVdRBfEi4fOq/ucXnmOOJNJnBl8Q1IW5NYwF0nL1sW2jNVnQsp6a0sjLs0QWHEs1J0OMyru0vsWCJfkFQn1ozla62EzKClNW0dAvRJcHsOCD2JWCzToM3HysghPkT1jAZvfrW1ndBDYCipetgSx9AoYb2jYxrxFx3BoZBd23zIy5WDa0V/HkwijV04GEt0/V/rXF3pQzEaKE07sY8rjeTHNew/bmmNcid3/MsPv/8596j6ir4QmuxaHi/kP5ZTQWjtjytUADB7HqU="
```

Comment:
can you elaborate on what these secure env vars are called, and what their purpose is? if we need to update them later, how can that be done?

Reply:
Ah, good callout. I meant to do this but forgot. I'll comment inline.

Reply:
I added the name of the variable in .travis.yml, then used the commit message to explain things. Anyone ending up git-blaming these should land in this commit: ecb728a.